### PR TITLE
Adds a version number to every dependency on org.apache.httpcomponent…

### PR DESCRIPTION
**Pull request by adrianbk**
…s:httpclient

### CODE REVIEW

**Comment by nik9000** [comment]
@rjernst this one is for you I think.

@adrianbk can you sign the [cla](https://www.elastic.co/contributor-agreement)? Its required for all changes.

File: plugins/discovery-ec2/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ esplugin {
 
 dependencies {
   compile 'com.amazonaws:aws-java-sdk-ec2:1.10.19'
-  compile 'org.apache.httpcomponents:httpclient'
+  compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
```
Comment (dadoonet) [inline plugins/discovery-ec2/build.gradle]: Warning on this one (and on s3). 
It might pull a wrong version.

AWS SDK defines it's own version

**Comment by adrianbk** [comment]
@nik9000 Yep, I'd signed the CLA before submitting the PR.

**Comment by adrianbk** [comment]
Here's the background for this PR: https://discuss.gradle.org/t/dependency-resolution-extremely-slow-with-s3-repositories/

**Comment by clintongormley** [comment]
CLA has been signed

File: plugins/discovery-ec2/build.gradle (hunk 1 above)
Comment (dadoonet) [inline plugins/discovery-ec2/build.gradle]: Forget my comment. Actually we are also pulling today version 4.3.6.

**Comment by rjernst** [comment]
Thank you @adrianbk!

**Pull request merged**